web/src/lib/components/user-settings-page/user-api-key-list.svelte

```diff
@@ -112,7 +112,7 @@
 <section class="my-4">
   <div class="flex flex-col gap-2" in:fade={{ duration: 500 }}>
     <div class="mb-2 flex justify-end">
-      <Button size="sm" on:click={() => (newKey = { name: 'API Key' })}>New API Key</Button>
+      <Button size="sm" on:click={() => (newKey ? (newKey.name = 'API Key') : '')}>New API Key</Button>
```

Comment:
what is this change?

Reply:
Reverted. I wanted it not use `Partial<APIKeyResponseDto>` but it seems it's way easier to use it